src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 import {Alert} from 'react-native';
 import {generateRandomId} from 'stream-chat-react-native-core';
-import {launchImageLibrary} from 'react-native-image-picker';
+import ImagePicker from 'react-native-image-crop-picker';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The library `react-native-image-crop-picker` is used instead of `react-native-image-picker`. This new library provides more features like cropping and editing the image before uploading. However, please ensure that necessary permissions are handled correctly as this library requires both camera and external storage access permissions.
```diff
- import {launchImageLibrary} from 'react-native-image-picker';
+ import ImagePicker from 'react-native-image-crop-picker';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->